ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -28,7 +28,8 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
-          trigger-phrase: ''
+          trigger-phrase: ''    
+          skip-build-phrase: 'qa tests .* skip build'
```

Comment:
Yeah i read that its broken too, but its also mid 2016 article, worth trying just to see. I will update the string, +1 for ^jenkins do not test(.*)' ( can be qa or doc changes)